x-pack/legacy/plugins/infra/server/lib/snapshot/response_helpers.ts

```diff
@@ -73,12 +73,13 @@ export const getIPFromBucket = (
   nodeType: InfraNodeType,
   bucket: InfraSnapshotNodeGroupByBucket
 ): string | null => {
-  const ip = get<typeof bucket, unknown>(
-    bucket,
-    `ip.hits.hits[0]._source.${IP_FIELDS[nodeType]}`,
-    null
-  );
-
+  const inventoryModel = findInventoryModel(nodeType);
+  if (!inventoryModel.fields.ip) {
+    return null;
+  }
+  const ip = get(bucket, `ip.hits.hits[0]._source.${inventoryModel.fields.ip}`, null) as
+    | string[]
```

Comment:
What does this leading `|` do? Is this the same as:
```ts
get(bucket, `ip.hits.hits[0]._source.${inventoryModel.fields.ip}`, null) as  string[] | null
```

Reply:
Uh... I think it's suppose to be written the way you have it. Looks like a typo to me that didn't get caught by the lint checker.

Reply:
OK... so when I try and fix it Prettier changes it back so it must be valid right? I mean you can't even change it.

Reply:
Sounds good haha. Prettier knows best.